src/main/java/hanieum/conik/domain/company/enumerate/CompanyStatus.java

```diff
@@ -0,0 +1,7 @@
+package hanieum.conik.domain.company.enumerate;
+
+public enum CompanyStatus {
+    PENDING,
+    APPROVED,
+    REJECTED
```

Comment:
지금 상태가 어떤 용도의 상태인지 알 수 있게 아래 처럼 바꾸는 건 어떤가요?

```suggestion
    REGISTER_PENDING,
    REGISTER_APPROVED,
    REGISTER_REJECTED
```